python/cudf/cudf/tests/utils.py

```diff
@@ -56,15 +64,18 @@ def assert_eq(a, b, **kwargs):
     functions.
     """
     __tracebackhide__ = True
+
     if hasattr(a, "to_pandas"):
         a = a.to_pandas()
     if hasattr(b, "to_pandas"):
         b = b.to_pandas()
+
+    update_multiindex_name(a, b)
```

Comment:
We don't want to do this here, we want to guarantee our names are correct instead of hiding it in the test equality checker.

Reply:
Test cases have been u[dated to take care of `None` name issue in MultiIndex.